icu4j/main/classes/core/src/com/ibm/icu/message2/Mf2DataModel.java

```diff
@@ -664,6 +664,17 @@ public Variable build() {
     @Deprecated
     public static class OrderedMap<K, V> extends LinkedHashMap<K, V> {
         private static final long serialVersionUID = -7049361727790825496L;
+
+        /**
+         * {@inheritDoc}
+         *
+         * @internal ICU 72 technology preview
+         * @deprecated This API is for ICU internal use only.
+         */
+        @Deprecated
+        public OrderedMap() {
```

Comment:
FYI Mihai also added this in https://github.com/unicode-org/icu/pull/2194
Looks like the change is exactly the same, so there is no conflict :-)